ts/apps/web/src/components/header.tsx

```diff
@@ -39,22 +41,57 @@ export default function Header() {
           >
             Roadmap
           </a>
+        </nav>
+
```

Comment:
this is very close but i would want the theme swticher to also make it in the drop down nav, and be removed from the header, once the screen is small enough